process_report/process_report.py

```diff
@@ -72,6 +97,12 @@ def main():
         nargs="+",
         help="One or more CSV files that need to be processed",
     )
+    parser.add_argument("--upload-to-s3", action="store_true")
```

Comment:
can you add help text saying that upload-to-s3 implies that the input csv reports will be fetched from s3? It's not just purely to upload to s3

Reply:
It should be a separate argument in fact.

Reply:
you are right, that'd be the best.